x/wasm/keeper/keeper_test.go

```diff
@@ -319,7 +319,7 @@ func TestInstantiate(t *testing.T) {
 
 	gasAfter := ctx.GasMeter().GasConsumed()
 	if types.EnableGasVerification {
-		require.Equal(t, uint64(0x142fa), gasAfter-gasBefore)
+		require.Equal(t, uint64(0x1402a), gasAfter-gasBefore)
```

Comment:
Why is the fee reduced?

Reply:
I think the cause is the decrease in the size of msg as pubkey is distinguished by type.